docs/rules.md

```diff
@@ -1,26 +1,96 @@
-# Default rules
+# Included validaton-rules
```

Comment:
@RickvdStaaij 

![img_2290](https://cloud.githubusercontent.com/assets/605483/17456649/771a297c-5bdf-11e6-9fa0-f04b2bdd70cb.JPG)

I'd like to buy an “I”!